spec/values/calculation/round/strategies/down.hrx

```diff
@@ -0,0 +1,197 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `round()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> strategy/down/step_is_multiple_of_number/input.scss
```

Comment:
You don't need `strategy/down` here; this is already in a `strategy/down` directory.